src/components/ApplicationListPage.tsx

```diff
@@ -9,10 +9,11 @@ import {
 } from '@openshift-console/dynamic-plugin-sdk';
 import { LoadingBox } from '@patternfly/quickstarts';
 
+import { fetchDataFrequency, pipelinesBaseURI } from '../const';
```

Comment:
Do you have extra commits here?  Looks like these changes in the src folder have already been merged.